changelog.txt

```diff
@@ -8,6 +8,7 @@ V.Next
 - [MINOR] Adds new API support for the broker - SSO token and flight support (#1290)
 - [MINOR] Elevates AndroidCommon Logger out of internal package (#1279)
 - [MINOR] Automate broker protocol version computation. (#1301)
+- [MAJOR] New API on KeyAccessor (#1309)
```

Comment:
Technically, by adding a method to an interface without a default implementation, this is a breaking change.